manifests/params.pp

```diff
@@ -142,11 +151,17 @@
       $pam_module_path     = '/usr/local/lib/openvpn/openvpn-auth-pam.so'
       $additional_packages = ['easy-rsa']
       $easyrsa_source      = '/usr/local/share/easy-rsa'
+      $default_easyrsa_ver = '2.0'
       $namespecific_rclink = true
       $systemd             = false
     }
     default: {
       fail("Not supported OS family ${::osfamily}")
     }
   }
+  $easyrsa_version = $facts['easyrsa'] ? {
+    undef   => $default_easyrsa_ver,
+    ''      => $default_easyrsa_ver,
```

Comment:
the fact should be reworked to only be undef or return a string.

Reply:
done